package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "express-api-skeleton",
+  "version": "1.0.0",
+  "description": "Skeleton for Express APIs.",
+  "main": "app.js",
+  "scripts": {
+    "start": "./node_modules/.bin/gulp run",
+    "lint": "./node_modules/.bin/gulp lint",
+    "test": "./node_modules/.bin/gulp test"
+  },
+  "pre-commit": [
+    "lint"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/osu-mist/express-api-skeleton.git"
+  },
+  "author": "osu-mist",
+  "license": "AGPL-3.0",
+  "bugs": {
+    "url": "https://github.com/osu-mist/express-api-skeleton/issues"
+  },
+  "homepage": "https://github.com/osu-mist/express-api-skeleton#readme",
+  "dependencies": {
+    "app-root-path": "^2.1.0",
+    "body-parser": "^1.18.3",
+    "capitalize": "^2.0.0",
+    "compose-middleware": "^5.0.1",
+    "config": "^2.0.1",
+    "deasync": "^0.1.14",
+    "decamelize": "^2.0.0",
+    "decode-uri-component": "^0.2.0",
+    "express": "^4.16.3",
+    "express-basic-auth": "1.1.7",
+    "express-openapi": "^3.1.0",
+    "express-winston": "^3.0.0",
+    "js-yaml": "^3.12.0",
+    "jsonapi-serializer": "^3.5.6",
+    "lodash": "^4.17.11",
+    "moment": "^2.22.2",
+    "query-string": "^6.2.0",
+    "simple-git": "^1.96.0",
+    "swagger-parser": "^6.0.3",
+    "winston": "^3.1.0",
+    "winston-daily-rotate-file": "^3.3.2"
```

Comment:
`oracledb` needs to be installed for your project.

Ref: https://github.com/osu-mist/express-api-skeleton#getting-data-source-from-the-oracle-database